.github/workflows/build.yml

```diff
@@ -29,13 +29,13 @@ jobs:
     - name: Get Short Sha
       id: short-sha
       run: echo "::set-output name=sha::$(git rev-parse --short=12 HEAD)"
-    - id: 'auth'
-        name: 'Authenticate to Google Cloud'
-        uses: 'google-github-actions/auth@v1'
-        with:
-          credentials_json: '${{ secrets.GCR_PUBLISH_KEY }}'
-      - name: Set up Cloud SDK
-        uses: google-github-actions/setup-gcloud@v0
+    - name: 'Authenticate to Google Cloud'
```

Comment:
Clarification: is the fix here the removal of the indentation or the switch in order with `name` coming before `id`?

Reply:
Essentially I didn't properly check the job before clicking merge on a different PR.
The formatting was a problem, and both are the fix.

This PR fixes the formatting essentially, and prepares us to use the secrets from `terraform-ap-deployment`